contracts/PvpArena.sol

```diff
@@ -526,7 +526,7 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         bool attackerWon;
     }
 
-    function createDuelist(uint256 id) internal returns (Duelist memory duelist) {
+    function createDuelist(uint256 id) internal view returns (Duelist memory duelist) {
```

Comment:
How can this be a viev if it mutates `duelist.basePower`, what am I missing?

Reply:
> How can this be a viev if it mutates `duelist.basePower`, what am I missing?

Changing to `view` indicates that the contract data is not changing.  The `duelist` is the return value, not a contract storage member.